### PR TITLE
MSTONKS --> inactive

### PR DESCRIPTION
**Pull request by papasnowee**

### CODE REVIEW

File: data/mainnet/tokens.js [hunk 1]
```diff
@@ -3108,7 +3108,8 @@ module.exports = {
   mirrorAAPL: {
     chain: CHAINS_ID.ETH_MAINNET,
     isDegen: true,
-    category: VAULT_CATEGORIES_IDS.MSTONKS,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
+    inactive: true,
     logoUrl: './icons/mirror-aapl.png',
     apyIconUrls: ['./icons/mir.png'],
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:3114]: Forgot to mention this before, but please also empty both these apy fields to just empty lists: `[],`, please also do it for the vaults you moved to inactive before. Right now the UI still shows the symbols for the rewards: 
image

File: data/mainnet/tokens.js [hunk 2]
```diff
@@ -3133,8 +3134,8 @@ module.exports = {
   mirrorAMZN: {
     chain: CHAINS_ID.ETH_MAINNET,
     isDegen: true,
-    category: VAULT_CATEGORIES_IDS.MSTONKS,
-    logoUrl: './icons/mirror-amzn.png',
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:3137]: Don't remove this line

File: data/mainnet/tokens.js [hunk 3]
```diff
@@ -3158,7 +3159,7 @@ module.exports = {
   mirrorGOOG: {
     chain: CHAINS_ID.ETH_MAINNET,
     isDegen: true,
-    category: VAULT_CATEGORIES_IDS.MSTONKS,
+    category: VAULT_CATEGORIES_IDS.MStoTONKS,
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:3162]: Think this is a mistake :)

File: package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "./run-standalone.sh",
     "dev": "nodemon src/index.js",
-    "test": "PORT=3030 MONGODB_DB_NAME=harvest-test-local ACTIVE_ENDPOINTS=all mocha ./test/integration/*  --timeout 600000",
+    "test": "mocha ./test/integration/*  --timeout 600000",
```
Comment (CryptJS13) [inline package.json:12]: Don't merge this change please

**Review by CryptJS13 — COMMENTED** [review]
Some comments, please also change the `"doHardwork"` setting to `"onlyProfit"` for all of them.

File: data/mainnet/tokens.js (hunk 2 above)
Comment (papasnowee) [reply data/mainnet/tokens.js:3137]: done

File: package.json (hunk 4 above)
Comment (papasnowee) [reply package.json:12]: I'm sorry, done

File: data/mainnet/tokens.js [hunk 5]
```diff
@@ -3159,7 +3160,7 @@ module.exports = {
   mirrorGOOG: {
     chain: CHAINS_ID.ETH_MAINNET,
     isDegen: true,
-    category: VAULT_CATEGORIES_IDS.MStoTONKS,
+    category: VAULT_CATEGORIES_IDS.MSTONKS,
```
Comment (CryptJS13) [inline data/mainnet/tokens.js]: It should be inactivated, like the rest

**Pull request merged**